projects/vdk-plugins/vdk-structlog/src/vdk/plugin/structlog/structlog_plugin.py

```diff
@@ -87,8 +216,33 @@ def vdk_configure(self, config_builder: ConfigurationBuilder):
             ),
         )
 
+        config_builder.add(
+            key=SYSLOG_URL,
+            default_value="localhost",
+            description="The hostname of the endpoint to which VDK logs will be sent through SysLog.",
+        )
+        config_builder.add(
+            key=SYSLOG_PORT,
+            default_value=514,
+            description="The port of the endpoint to which VDK logs will be sent through SysLog.",
+        )
+        config_builder.add(
+            key=SYSLOG_ENABLED,
+            default_value=False,
+            description="If set to True, SysLog log forwarding is enabled.",
+        )
+        config_builder.add(
+            key=SYSLOG_SOCK_TYPE,
+            default_value="UDP",
+            description=f"Socket type for SysLog log forwarding connection. Currently possible values are "
+            f"{list(SYSLOG_SOCK_TYPE_VALUES_DICT.keys())}",
+        )
+
     @hookimpl
     def vdk_initialize(self, context: CoreContext):
+        # Use this plugin for logging config and warn the user about the vdk-core logging deprecation
+        os.environ["VDK_USE_STRUCTLOG"] = "1"
+        configure_initial_logging()
```

Comment:
Can you please post log outputs of jobs using all formats, e.g. one for json, one fore console, one for ltsv? 

I have a hunch that this might mess up parts of the initial log. But I also see we still eject the formatter later on, so not sure. It would be nice to see the full output.

Reply:
Added 1 for JSON and 1 for console to the PR description.

Reply:
Here it is: https://pastebin.com/8NnfF0En